blocks/library/latest-posts/index.js

```diff
@@ -19,11 +22,13 @@ registerBlockType( 'core/latestposts', {
 
 	defaultAttributes: {
 		poststoshow: 5,
+		displayPostDate: false,
 	},
 
-	edit: class extends wp.element.Component {
+	edit: withInstanceId( class extends wp.element.Component {
```

Comment:
We're moving way from `wp.*` as much as we can in favor of `import { Component } from 'element';`